### PR TITLE
[bitnami/postgresql] fix postgresql.replicationPasswordKey secret key logic

### PR DESCRIPTION
**Pull request by AdrianAxelsson**
Signed-off-by: Adrian Axelsson <adrian.axelsson@hotmail.se>

### Description of the change

When deploying the current version of the postgresql chart with replication, without specifying an existing secret, postgresql.replicationPasswordKey will not default to "replication-password". 
This change fixes the if/else logic

### Benefits

The chart can be installed successfully with architecture=replication without having to specify an existing secret

### Applicable issues

  - fixes #

### Checklist

- [X] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [X] Title of the pull request follows this pattern [bitnami/<name_of_the_chart>] Descriptive title
- [X] All commits signed off and in agreement of [Developer Certificate of Origin (DCO)](https://github.com/bitnami/charts/blob/master/CONTRIBUTING.md#sign-your-work)

### CODE REVIEW

File: bitnami/postgresql/templates/_helpers.tpl [hunk 1]
```diff
@@ -108,9 +108,9 @@ Get the replication-password key.
         {{- printf "%s" (tpl .Values.global.postgresql.auth.secretKeys.replicationPasswordKey $) -}}
     {{- else if .Values.auth.secretKeys.replicationPasswordKey -}}
         {{- printf "%s" (tpl .Values.auth.secretKeys.replicationPasswordKey $) -}}
-    {{- else -}}
-        {{- "replication-password" -}}
     {{- end -}}
+{{- else -}}
+    {{- "replication-password" -}}
```
Comment (marcosbc) [inline bitnami/postgresql/templates/_helpers.tpl:113]: Shouldn't we also keep the `else` condition that was removed above, in this PR? If we have both, we can basically leave "replication-password" as a fallback value whenever something is not properly configured.

**Review by marcosbc — CHANGES_REQUESTED** [review]
Thanks for the contribution! Please have a look at my comments.

File: bitnami/postgresql/templates/_helpers.tpl (hunk 1 above)
Comment (AdrianAxelsson) [reply bitnami/postgresql/templates/_helpers.tpl:115]: Good thinking, I've added the `else` condition again.

Comment (ComputerBaer) [reply bitnami/postgresql/templates/_helpers.tpl:115]: But then it should be done the same way for `adminPasswordKey` and `userPasswordKey`, right?

**Review by marcosbc — APPROVED** [review]
LGTM!

**Pull request merged**